selenium-tests/webpack.common.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     filename: '[name].js',
     sourceMapFilename: '[file].map',
   },
-  target: ['web', 'es5'],
```

Comment:
Is ES5 no longer needed here? Or, why was it there in the first place?

Reply:
I'm not sure why this was there/worked before; according to the webpack docs, `es5` is not a valid string:
https://v4.webpack.js.org/configuration/target/#string